Templates/Android/generated/BaseMethodCollectionPage.java.tt

```diff
@@ -5,12 +5,26 @@
 <#host.TemplateName = BaseTypeCollectionPage(c);#>
 <#=writer.WriteHeader()#>
 <#=CreatePackageDef(host)#>
+<# 
+	var deltaPage = false;
+	if (TypeName(c).Length > 6 && TypeName(c).Substring(TypeName(c).Length-5) == "Delta") {
```

Comment:
Per conversation, and as far as I recall, you mentioned that this is used to determine whether the collection page is a delta page as there are different types of delta pages. This looks like it is using some convention where "Delta" is suffix of name. Could you add comments describing what this is in case we need to revisit this?